toucan_connectors/toucan_connector.py

```diff
@@ -258,12 +259,20 @@ class ToucanConnector(BaseModel, metaclass=ABCMeta):
     type: str = Field(None)
     secrets_storage_version = Field('1', **{'ui.hidden': True})
 
+    # Default ttl for all connector's queries (overridable at the data_source level)
+    # /!\ cache ttl is used by the caching system which is not implemented in toucan_connectors.
+    cache_ttl: Optional[int] = Field(None, title='TTL (cache')
```

Comment:
```suggestion
    cache_ttl: Optional[int] = Field(None, title='TTL (cache)', **{'ui.hidden': True})
```
Ca complètement simplement la valeur du "title" et ça cache le champs à l'utilisateur sur le formulaire de configuration.
Sauf si on souhaite que l'utilisateur final puisse configurer la durée de cache (de la même manière que le Retry)

Reply:
@f-bb-toucantoco yes on veut laisser la possibilité au concepteur de spécifier les TTL qu'il veut, par datasource (mais on aura quand même une valeur par défaut : une pour notre mongo (genre 10min) et une pour les autres connecteurs (genre 1h), l'idée étant que le cache de mongo soit + volatile pour ne pas trop evict le cache des connecteurs)

Reply:
Je suis en accord avec toi.
Il faut sûrement prévoir une petite communication côté produit sur l'ajout du champs (il y a des sujets de refactorisation des formulaires en cours et ça peut (de pas beaucoup) impacter le design notamment)